nservicebus/sagas/timeouts.md

```diff
@@ -34,13 +34,13 @@ Multiple timeouts can be requested when processing a message. The individual tim
 
 snippet: saga-multiple-timeouts
 
-NOTE: Sending a timeout with the same data multiple times will result in that timeout being processed multiple times by the saga.
+NOTE: As timeouts are just messages sending a multiple timeouts with the same state will result in that timeout being processed multiple times by the saga.
 
 ## Revoking timeouts
 
 A timeout that has been scheduled cannot be revoked. This means that when the timeout timestamp has elapsed then this timeout message will be queued and then processed.
 
-A timeout is a regular message and once requested, a timeout message can already be in transit or queued. Once that has happened there is no way to revoke (delete) a timeout. It is common to perform a state check in a timeout handler to see if the timeout is still applicable for processing.
+A timeout is a regular message and once requested, a timeout message can already be in transit or queued. Once that has happened there is no way to revoke (delete) a timeout. Sending a timeout will not invalidate/revoke any prior sent timeout. It is common to perform a state check (e.g. comparing a version number, a timestamp or a status field) in a timeout handler to see if the timeout is still applicable for processing. 
```

Comment:
In the context of the above suggestion, I don't think anything needs to change in this section. It can remain purely about revoking timeouts, and all it needs to say is that it cannot be done.

```suggestion
A timeout is a regular message and once requested, a timeout message can already be in transit or queued. Once that has happened there is no way to revoke (delete) a timeout. It is common to perform a state check in a timeout handler to see if the timeout is still applicable for processing. 
```